src/main/kotlin/org/jetbrains/research/ml/coding/assistant/solutionSpace/SolutionSpaceEdge.kt

```diff
@@ -0,0 +1,9 @@
+package org.jetbrains.research.ml.coding.assistant.solutionSpace
+
+import com.github.gumtreediff.actions.model.Action
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.builder.WeightedEdge
+
+class SolutionSpaceEdge(
```

Comment:
Maybe it is better to create the `models` file and add `SolutionSpaceEdge`, `SolutionSpaceVertex`, `SolutionSpaceEdgeFactory`?